### PR TITLE
Storage Limit Update V0.12.5

### PR DESCRIPTION
**Pull request by janezpodhostnik**
DO NOT MERGE: flow-go version needs to be pinned to a tag first

References: https://github.com/dapperlabs/flow-go/issues/5127, https://github.com/onflow/flow-go/pull/208

## Description

Default behaviour has not changed.

Using the `--storage-limit` emulator flag storage limiting can be turned on. This changes the following:

- every account has a minimum Flow balance. If the account is below the minimum balance any transaction that touches that account will fail.
- if accounts storageUsed is more or equal to accounts storageCapacity at the end of a transaction that touches that account the transaction will fail.
- creating a new account deducts flow from the payer of that accounts creation.

______

For contributor use:

- [x] Targeted PR against `v0.12.5` branch
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work
- [x] Code follows the [standards mentioned here](https://github.com/onflow/flow-emulator/blob/master/CONTRIBUTING.md#styleguides)
- [x] Updated relevant documentation
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Added appropriate labels

### CODE REVIEW

**Comment by janezpodhostnik** [comment]
@turbolent where should I put the documentation for this emulator flag?

**Comment by turbolent** [comment]
@janezpodhostnik I think the README is the best place for now. agree @psiemens?

**Review by turbolent — COMMENTED** [review]
@janezpodhostnik v0.12.5 is already released. Can you please branch off of it as v0.12.6 and target this branch against it?

File: blockchain.go [hunk 1]
```diff
@@ -397,14 +412,30 @@ func bootstrapLedger(
...
 
 	return nil
 }
 
+func configureBootstrapProcedure(ctx fvm.Context, flowAccountKey flowgo.AccountPublicKey, supply cadence.UFix64) *fvm.BootstrapProcedure {
+	if ctx.LimitAccountStorage {
+		return fvm.Bootstrap(flowAccountKey,
+			fvm.WithInitialTokenSupply(supply),
+			fvm.WithAccountCreationFee(10000000),
+			fvm.WithMinimumStorageReservation(10000000),
```
Comment (turbolent) [inline blockchain.go:430]: Ideally these should match the values used on real networks by default, with the option to maybe configure them (like the computation limit). Maybe declare in and re-use constants from FVM

**Review by turbolent — APPROVED** [review]
Nice work!

File: blockchain.go (hunk 1 above)
Comment (janezpodhostnik) [reply blockchain.go]: done

**Review by turbolent — APPROVED** [review]
Great work, looks good!

**Comment by turbolent** [comment]
Closing, we'll merge this feature directly to master instead of backporting it to the older/current version

**Pull request closed without merging**